examples/plotter/Cargo.toml

```diff
@@ -15,7 +15,7 @@ name = "plotter"
 
 [dependencies]
 slint = { path = "../../api/rs/slint" }
-plotters = { version = "0.3.4", default-features = false, features = ["bitmap_backend", "surface_series"] }
+plotters = { version = "0.3.4", default-features = false, features = ["bitmap_backend", "surface_series", "fontconfig-dlopen"] }
```

Comment:
plotters depends against font-kit which depends against an old version of yesconfig-fontconfig-sys

So This dependency in our example means that we  cannot upgrade our version of yesconfig-fontconfig-sys

Reply:
Yes :-(

I can see three options:

1) Change the example to use the dummy wasm backend that omits text altogether (and rename the backend, of course)
2) Implement the backend by means of fontdb and tiny-skia and draw text ourselves
3) Wait until upstream versions are up-to-date

(of course we can also stage this, do (1) first, then (2))